SemanticUI/Collections/Breadcrumb.elm

```diff
@@ -0,0 +1,85 @@
+module SemanticUI.Collections.Breadcrumb exposing
+  ( Config
+  , BreadcrumbItem
+  , init
+  , size
+  , divider
+  , breadcrumb
+  , breadcrumbPlain
+  , asActive
+  )
+{- Provides [breadcrumb](https://semantic-ui.com/collections/breadcrumb.html) collection.
+
+Usage example:
+
+   import SemanticUI.Collections.Breadcrumb as Breadcrumb
+
+   Breadcrumb.breadcrumbPlain
+     Breadcrumb.init
+     [ text "First", text "Second" ]
+-}
+
+import Html exposing (..)
+import Html.Attributes exposing (..)
+import SemanticUI
+
+
+{-| The configuration of a breadcrumb.
+-}
+type alias Config msg =
+    { divider : Html msg
+    , size : SemanticUI.Size
+    }
+
+{-| The simplest configuration of a breadcrumb. Corresponds to `class="ui breadcrumb"`.
+-}
+init : Config msg
+init =
+    { divider = text "/"
+    , size = SemanticUI.Medium
+    }
+
+{-| Specify the size of a breadcrumb.
+-}
+size : SemanticUI.Size -> Config msg -> Config msg
+size a model =
+    { model | size = a }
+
+{-| Specify the size of a breadcrumb.
+-}
+divider : Html msg -> Config msg -> Config msg
+divider a model =
+    { model | divider = a }
+
+{-| A single breadcrumb with contents and possible "active" state
+-}
+type alias BreadcrumbItem msg =
+    { isActive : Bool
+    , content : Html msg
+    }
+
+asActive : BreadcrumbItem msg -> BreadcrumbItem msg
+asActive model =
+    { model | isActive = True }
+
+{- view a breadcrumb -}
+breadcrumb : Config msg -> List (BreadcrumbItem msg)  -> Html msg
+breadcrumb cfg items = view cfg items
+
+{- view a breadcrumb with chosen defaults (not active)-}
+breadcrumbPlain : Config msg -> List (Html msg) -> Html msg
+breadcrumbPlain cfg items = view cfg (List.map (\item -> BreadcrumbItem False item) items)
```

Comment:
This transformation doesn't seem necessary. Caller can just produce the right list anyway, which is easier with `activeItem` and `inactiveItem` functions.

Reply:
It's mostly a convenience function to skip whole `isActive` deal and use defaults. Otherwise you have to create a `Breadcrumb` for each item, maybe there could be a `initBreadcrumb` that would get partially applied with `False`.

Reply:
@ocharles how would you prefer to handle items, separate module as with other components?